packages/client/src/routes.ts

```diff
@@ -9,3 +9,19 @@ export const enum ROUTES {
   ERROR_404 = '/not-found',
   DOCUMENTAION = '/documentation',
 }
+
+export function isRoute(path: string) {
+  return (
+    [
+      ROUTES.MAIN,
+      ROUTES.GAME,
+      ROUTES.FORUM,
+      ROUTES.LEADER_BOARD,
+      ROUTES.PROFILE,
+      ROUTES.AUTH,
+      ROUTES.REGISTER,
+      ROUTES.ERROR_404,
+      ROUTES.DOCUMENTAION,
+    ] as string[]
+  ).includes(path)
+}

```

Comment:
[Предложение]
Здесь можно избавиться от `as`, если вынести массив в переменную вот так:
```suggestion
export function isRoute(path: string) {
    const routes: string[] = [
        ROUTES.MAIN,
        ROUTES.GAME,
        ROUTES.FORUM,
        ROUTES.LEADER_BOARD,
        ROUTES.PROFILE,
        ROUTES.AUTH,
        ROUTES.REGISTER,
        ROUTES.ERROR_404,
        ROUTES.DOCUMENTAION,
    ];
    return routes.includes(path);
}
```

`as` просто выключает TS, лучше его не использовать, если это возможно

Reply:
исправила